modules/scanning-images.adoc

```diff
@@ -0,0 +1,86 @@
+// Module included in the following assemblies:
+//
+// * operating/examine-images.adoc
+:_module-type: CONCEPT
+[id="scanning-images_{context}"]
+= Scanning images
+
+[role="_abstract"]
+Central submits image scanning requests to Scanner.
+Upon receiving these requests, Scanner pulls the image layers from the relevant registry, checks the images, and identifies installed packages in each layer.
+Then it compares the identified packages and programming language-specific dependencies with the vulnerability lists and sends information back to Central.
+
+You can also integrate with another vulnerability scanner.
```

Comment:
This is missing a subject. Should it be 'You can also integrate it with. . .' or 'You can also integrate the Stackrox Scanner with another vulnerability scanner'?